pkg/csi/cinder/openstack/openstack.go

```diff
@@ -39,6 +39,7 @@ func AddExtraFlags(fs *pflag.FlagSet) {
 }
 
 type IOpenStack interface {
+	CheckBlockStorageAPI() (bool, error)
```

Comment:
What's the intention of this `bool` type value? I didn't see it's used anywhere. Isn't `error` sufficient?

Reply:
you are right, not used. Ill update